llvm/include/llvm/IR/Metadata.h

```diff
@@ -1840,6 +1842,15 @@ class NamedMDNode : public ilist_node<NamedMDNode> {
 // Create wrappers for C Binding types (see CBindingWrapping.h).
 DEFINE_ISA_CONVERSION_FUNCTIONS(NamedMDNode, LLVMNamedMDNodeRef)
 
+// Check if a given MDNode is a valid genaralized type metadata node.
+inline bool hasGeneralizedMDString(const MDNode *MD) {
```

Comment:
This shouldn't be an inline function in the header.

Reply:
This is now back as part of MDNode API and no inline.